ApeWorX/ape
# feat: support configuring test account initial balance

### What I did

This PR adds support for configuring test account initial balances from genesis using the `test:` config.
Both eth-tester and geth-dev honor this config.

Unblocks https://github.com/ApeWorX/ape/issues/264

releated:

hh: https://github.com/ApeWorX/ape-hardhat/pull/175
anvil: https://github.com/ApeWorX/ape-foundry/pull/105

### How I did it

Add the config
honor it in the providers

### How to verify it

connect to both test providers and check the balance(s) of the test accounts, it should be what you set it to.
You can also use currency strings in the config

### Checklist
